docs/content/how-to/aws/disaster-recovery.md

```diff
@@ -38,14 +57,46 @@ Regarding the Workers nodes assigned to the cluster, during the migration they w
 !!! important
     Keep in mind that this **"migration"** capability is only for **disaster recovery purposes**, please **DO NOT** use this to perform clusters migrations as a common task in your platform.
 
-!!! note
-    One considerations we should have is, both Hypershift deployments Source Management cluster and Destination Management cluster, had to have the `--external-dns` flags, in order to maintain the API server URL with the same entry. If that's not the case, the HostedCluster could not be migrated.
+- Hypershift Deployment access endpoints arguments
```

Comment:
Also what is this section with arguments? is this requirements for the process or is it something else?

Reply:
it's not part of the process itself but It reflects the different parameters required to deploy Hypershift and depending how they did installed the operator and deployed the cluster, they will need to avoid some steps on the real migration process.

Reply:
can we clarify that on the doc?